parquet-hadoop/src/test/java/org/apache/parquet/format/converter/TestParquetMetadataConverter.java

```diff
@@ -54,13 +32,43 @@
 import org.apache.parquet.format.RowGroup;
 import org.apache.parquet.format.SchemaElement;
 import org.apache.parquet.format.Type;
+import org.apache.parquet.hadoop.metadata.BlockMetaData;
+import org.apache.parquet.hadoop.metadata.ColumnChunkMetaData;
+import org.apache.parquet.hadoop.metadata.ColumnPath;
+import org.apache.parquet.hadoop.metadata.CompressionCodecName;
+import org.apache.parquet.hadoop.metadata.ParquetMetadata;
 import org.apache.parquet.schema.MessageType;
 import org.apache.parquet.schema.OriginalType;
 import org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName;
 import org.apache.parquet.schema.Type.Repetition;
 import org.apache.parquet.schema.Types;
+import org.junit.Assert;
+import org.junit.Test;
 
-import com.google.common.collect.Lists;
+import java.io.ByteArrayInputStream;
```

Comment:
Could you revert the non-functional import changes? I know it's annoying, but we try to keep these to a minimum to avoid conflicts when merging.